core/services/relay/evm/types/types.go

```diff
@@ -215,6 +221,10 @@ type RelayConfig struct {
 	// LLO-specific
 	LLODONID      uint32        `json:"lloDonID" toml:"lloDonID"`
 	LLOConfigMode LLOConfigMode `json:"lloConfigMode" toml:"lloConfigMode"`
+
+	// DualTransmission specific
+	EnableDualTransmission bool                   `json:"enableDualTransmission" toml:"enableDualTransmission"`
+	DualTransmissionConfig DualTransmissionConfig `json:"dualTransmission" toml:"dualTransmission"`
```

Comment:
Don't have great context, but wouldn't we want to make this config a pointer to allow nil values?

Reply:
And perhaps add the `config` suffix to the jobspec?

Reply:
@dimriou Changed the `DualTransmissionConfig` to a pointer in [`e8c9963` (#15202)](https://github.com/smartcontractkit/chainlink/pull/15202/commits/e8c9963344e781fbb2d4fe909c6f196bb188bfe6).